pysql_beam/examples/.ipynb_checkpoints/ppmktd-filter-effective-date-checkpoint.sql

```diff
@@ -0,0 +1,3 @@
+SELECT COMPANY, DIVISION, MARKET, COMMODITY, EFFECTIVE_DATE, ISSUE_DATE, DESCRIPTION, LOW_PRICE, HIGH_PRICE, PRICE_UM, CREATEDATE, CREATETIME, CREATEUSER, CREATESTATION, LASTDATE, LASTTIME, LASTUSER, LASTSTATION
```

Comment:
Do we need this file? This might be something specific to your personal or professional work.